baybe/exceptions.py

```diff
@@ -50,3 +50,7 @@ class DeprecationError(Exception):
 
 class UnidentifiedSubclassError(Exception):
     """A specified subclass cannot be found in the given class hierarchy."""
+
+
+class ModelNotTrainedError(Exception):
+    """A prediction/transformation is attempted before the model has been trained."""
```

Comment:
Does "transformation" also refer to a "Model"? If not, then this description should be changed.

Reply:
That version came from @Scienfitz, so will let the two of you discuss

Reply:
Look eg at sklearn where before using a `transform` you have to call `fit`

Reply:
And yes: "transformation" also does refer to the model, because it's part of the trained model. So I'll simply resolve for now, but feel free to reopen